CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: A new cosmetic option for adding an in-game death counter to the HUD.
 - Added: Exporting with a custom path now checks for conflicts with the input path.
 
+#### Logic Database
+
+- Added: Safely get the Energy Part in Golzuna Tower with Speed Booster and Spin Boost in Door Lock Rando.
```

Comment:
I wouldn't include this one right now considering door rando is not stable yet.

Reply:
well, if door rando is looking stable, maybe include this after all?

Reply:
If https://github.com/randovania/randovania/pull/4148 gets merged then this changeset becomes entirerly redundant, which would be fine for me. Or I guess the only thing left in this changeset would be the removal of the TODO comment about charging speed through the eye door.

Reply:
Since that pr was merged, I'd this one still relevant?

Reply:
I changed this PR so that all it does is to change the comment, that really aught to have been a "no door lock rando" condition before, but is fine now logically. This PR is relevant iff it's relevant to fix that comment.